packages/react/src/components/Combobox/Combobox.tsx

```diff
@@ -1,169 +1,489 @@
+        });
+        break;
+      case 'ArrowUp':
+        event.preventDefault();
+        setActiveIndex((prevActiveIndex) => {
+          if (prevActiveIndex === null) {
+            return filteredItems.length - 1;
+          }
+
+          // loop - if first item, go to last item
+          if (prevActiveIndex === 0) {
+            return filteredItems.length - 1;
+          }
+
+          return Math.max(prevActiveIndex - 1, 0);
+        });
+        break;
+      case 'Enter':
+        event.preventDefault();
+        if (activeIndex !== null && filteredItems[activeIndex]) {
+          const child = filteredItems[activeIndex];
+          if (React.isValidElement(child) && child.type === ComboboxItem) {
+            const props = child.props as ComboboxItemProps;
+            let item;
+            for (const valueItem of values) {
+              if (valueItem.value === props.value) {
+                item = valueItem;
+                break;
+              }
+            }
+            handleSelectItem(item as ValueItemType);
+          }
+        }
+        break;
+
+      case 'Backspace':
+        if (inputValue === '' && multiple && activeValues.length > 0) {
+          setActiveValues((prev) => prev.slice(0, prev.length - 1));
+        }
+        break;
+
+      default:
+        break;
+    }
+  };
+
+  return (
+    <ComboboxContext.Provider
+      value={{
+        size,
+        values,
+        activeValues,
+        multiple,
+        showEmptyChild,
+        activeIndex,
+        disabled,
+        readOnly,
+        label,
+        description,
+        hideLabel,
+        open,
+        inputRef,
+        refs,
+        inputValue,
+        activeDescendant,
+        error,
+        formFieldProps,
+        name,
+        setInputValue,
+        setActiveIndex,
+        handleKeyDown,
+        setOpen,
+        getReferenceProps,
+        setActiveValues,
+        /* Recieves index of item, and the ID of the button element */
+        setActiveItem: (index: number, id: string) => {
+          if (readOnly) return;
+          if (disabled) return;
+          setActiveIndex(index);
+          setActiveDescendant(id);
+        },
+        /* Recieves the value of the item, and searches for it in our values lookup */
+        onItemClick: (value: string) => {
+          if (readOnly) return;
+          if (disabled) return;
+          let item;
+          for (const valueItem of values) {
+            if (valueItem.value === value) {
+              item = valueItem;
+              break;
+            }
+          }
+          handleSelectItem(item as ValueItemType);
+        },
+      }}
+    >
+      <Box
+        className={classes.combobox}
+        ref={portalRef}
+      >
+        {/* This is only for the Combobox to work in forms */}
+        {name && <ComboboxNative />}
+
+        <ComboboxLabel />
+        <ComboboxInput />
+        {formFieldProps.hasError && <ComboboxError />}
```

Comment:
Conditional rendering breaks `aria-live` so I you have to bring up the div with it here. Maybe just drop the `ComboboxError` for then?